wiki/Tournaments/UKCC/X/en.md

```diff
@@ -6,13 +6,610 @@ tags:
+- We take no-shows seriously. If you fail to show up to your match without valid reason, you may forfeit your right to participate in future UKCC tournaments.
+- The referee cannot play in the match.
+
+### Qualifiers
+
+- Players will play each map in the Qualifiers mappool once.
+- There will be 3 NoMod, 2 Hidden, 2 Hard Rock and 2 Double Time maps.
+- The top 32 scores will progress into the Swiss bracket.
+
+### Swiss
+
+- Since Swiss bracketing is a natural seeding system, matchups will be drawn between **top 16 and bottom 16 players randomly** to begin.
+- Two matches will happen every weekend for each player, one on Saturday and one on Sunday - therefore, **two matchups will be drawn per weekend** for each player.
+- The order the matches are played in **does not matter**.
+- Matchups will be drawn between players with the **same aggregate match W-L-D score** (eg. players at 3-0-1 will prioritise playing against the players with the most similar W-L-D), and will prioritise **avoiding rematches** until each active player has played each other once.
+
+### Rolling
+
+- When both players are in the lobby and ready, the referee will ask the players to `!roll`. The order of picks of bans is decided using the [UKCC 4 roll system](https://i.imgur.com/S0VIvni.png).
+
+### Banning
+
+- Each player will have **one** ban.
+- Players have **2 minutes** to ban a map. Failure to ban a map within this time limit results in a lost ban.
+- There are no restrictions on bans and picks. Players are free to ban/pick any map regardless of mod.
+
+### Warmups
+
+- There will be **no** warmups played.
+
+### Picking
+
+- Players have **90 seconds** to pick a map. Failure to pick a map within this time limit results in pick choice being passed to the opposing player.
+- **Failed scores will count, and every map will be played with the NF modification enabled**
+- Each player picks **four times**. Once 8 maps have been picked and played, the match ends.
+- There are **no Tiebreakers.**
+
+### Health system
+
+- Each player who qualifies into Swiss will **begin with 8,000,000 HP**, which carries through matches.
+- During a match, the **losing player of each map pick** will **lose HP equal to the score difference** between the two players.
+- If a player’s HP reaches 0, they are **immediately eliminated from the tournament**, and if their match is still in progress it ends immediately.
+
+### Pocket Picks
+
+- When signing up to the tournament, each player will **submit three maps** to be listed as “Pocket Picks” for use during Swiss.
+- Pocket Picks can be **any map, with any mods**. The mods chosen are forced.
+- While there is room to push boundaries, the **rule of common sense** applies when submitting Pocket Picks - the maps will be reviewed case by case, and if a map is too far out of the playable range it will be rejected and a new Pocket Pick will have to be submitted.
```

Comment:
```suggestion
- While there is room to push boundaries, the **rule of common sense** applies when submitting Pocket Picks—the maps will be reviewed case by case, and if a map is too far out of the playable range it will be rejected and a new Pocket Pick will have to be submitted.
```